### ISSUE TITLE
Feature request - body height attribute to control a scrollable text field

### ISSUE THREAD

**Issue by IndigoJay** [sweatybridge/streamlit-text-label#13]
I am using `streamlit-text-label` to annotate labels on documents. Frequently the body text is multiple pages long, and the users use the browser's scrollbar to navigate, however the Label buttons remain at the top and the Update button at the bottom. When documents are multiple pages long the UI becomes hard to manage. 

I looked into StreamLit scrollable containers [which is not yet a feature](https://github.com/streamlit/streamlit/issues/2169), and I looked at various [text](https://discuss.streamlit.io/t/scrolling-text-containers/26485) and [html](https://stackoverflow.com/questions/74185715/is-there-a-way-to-make-a-vertically-scrollable-list-of-elements-in-streamlit) alternatives, and they do not appear compatible with streamlit-text-label.

This feature request is to add a `height` attribute to make the body text section scrollable. 

Mock-up:

`selections = label_select(body="lorem ipsum", labels=["Noun", "Verb"], height=50)`

![Screenshot from 2023-05-22 16-08-35](https://github.com/sweatybridge/streamlit-text-label/assets/30580858/d7bf1c9a-cae0-43f6-83d9-2a8ab6d060e5)

**Comment by IndigoJay** [comment]
Label Studio has this:

https://labelstud.io/templates/text_summarization.html#Enhance-this-template

> ###  Display long text samples with a scrollbar
>
> If you want to change how Label Studio displays long text samples on the labeling interface, you can use the [View](https://labelstud.io/tags/view.html) tags to wrap labeling tags with CSS styling.
> 
> For example, you can constrain the text sample to a specific height, making it easier to keep the text summary that annotators provide visible.

```
xml
<View style="height: 300px; overflow: auto;">
    <Text name="text" value="$longText" />
</View>
```

**Comment by sweatybridge** [comment]
I've just released 0.1.2 with your changes. Glad that you are finding this package useful.
